rig/machine_control/sdram_file.py

```diff
@@ -0,0 +1,84 @@
+class SDRAMFile(object):
```

Comment:
File missing introductory docstring.

    """File-like interface wrapper for SDRAM of SpiNNaker chips."""